code/modules/virus2/effect/stage_4.dm

```diff
@@ -79,7 +82,7 @@
 	name = "Vox Pox"
 	desc = "A previously experimental syndrome that found its way into the wild. Causes the infected to mutate into a Vox."
 	stage = 4
-	badness = 2
+	badness = 4
```

Comment:
Life as a Vox is 5-level bad if you ask me. I would also make them disease-infected, similar to how the cursed dice is supposed to make you into a disease-ridden vox.

Reply:
Technically they're already disease infected for that effect to proc in the first place :^)
But fine, I guess it deserves to be level 5.  I'll take care of preset and stuff like mid-round outbreaks next part.